### PR TITLE
Replicator docs and tutorial

### PR DESCRIPTION
**Pull request by arrawatia**
@samjhecht @cotedm @gwenshap @theduderog Can you please review the tutorial ? 

The images are not uploaded yet, so to run through the tutorial you will need to build the images locally from the 3.1.x branch. Even if you don't run through the tutorial, comments on the content would be helpful :) .

### CODE REVIEW

**Comment by jonchiu** [comment]
Ran through the tutorial, looks great!

File: docs/tutorials/kafka-connect-replicator.rst [hunk 1]
```diff
@@ -0,0 +1,402 @@
+.. _kafka_connect_replicator :
+
+Kafka Connect Replicator
+--------------------
+
+In this section, we provide a tutorial for running Kafka Connect Replicator which replicates data from two source Kafka clusters to a destination Kafka cluster.  By the end of this tutorial, you will have successfully run Kafka Connect Replicator and replicated  data for two topics from different source clusters to a destination cluster.
```
Comment (cotedm) [inline docs/tutorials/kafka-connect-replicator.rst]: nit: a couple of inadvertent double spaces here

File: docs/tutorials/kafka-connect-replicator.rst [hunk 2]
```diff
@@ -0,0 +1,402 @@
...
+
+
+It is worth noting that we will be configuring Kafka and Zookeeper to store data locally in the Docker containers.  For production deployments (or generally whenever you care about not losing data), you should use mounted volumes for persisting data in the event that a container stops running or is restarted.  This is important when running a system like Kafka on Docker, as it relies heavily on the filesystem for storing and caching messages.  Refer to our `documentation on Docker external volumes <operations/external-volumes.html>`_ for an example of how to add mounted volumes to the host machine.
```
Comment (cotedm) [inline docs/tutorials/kafka-connect-replicator.rst]: Probably should be a little more formal here "For production deployments (or generally whenever you care about not losing data)"
maybe change to "For deployments that require persistent data (e.g. production deployments)"

File: docs/tutorials/kafka-connect-replicator.rst [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+
+  .. sourcecode:: bash
+
+    docker run \
+      --net=host \
+      --rm \
+      confluentinc/cp-kafka:3.1.0-SNAPSHOT \
+      bash -c "seq 1000 | kafka-console-producer --request-required-acks 1 --broker-list localhost:9082 --topic bar && echo 'Produced 1000 messages.'"
+
+  Now lets ``exec`` into the Kafka Connect container and run the replicator connector. Enter the following commands on your terminal. You should see output simialar to step 6 above.
```
Comment (cotedm) [inline docs/tutorials/kafka-connect-replicator.rst]: simialar --> similar

Comment (arrawatia) [inline docs/tutorials/kafka-connect-replicator.rst]: @samjhecht can you please add this while you are wordsmithing the contents :).

**Comment by arrawatia** [comment]
@cotedm 
1. No magic. They are all assigned different ports in the docker-compose file.
2. The second replicator is to showcase the multiple cluster use-case, I will add a sentence which says that. Your comment about explicitly saying that each topic does not require a separate replicator instance makes sense, I will add that.

**Comment by samjhecht** [comment]
This has been duplicated elsewhere.  closing.

**Pull request closed without merging**